.github/workflows/test.yml

```
@@ -126,7 +126,7 @@ jobs:
       - name: Install rust
         uses: actions-rust-lang/setup-rust-toolchain@v1
```

These changes include the ones in https://github.com/OpenZeppelin/rust-contracts-stylus/pull/427.

Include this comment in the workflow file for future reference, and we can close my PR.

```suggestion
        # We run in nightly to make use of the `#[coverage(off)]` attribute (see _contracts/src/lib.rs_).
        # We'll return to stable release after the tracking issue is merged (and the feature is stable).
        # See: https://github.com/rust-lang/rust/issues/84605
        uses: actions-rust-lang/setup-rust-toolchain@v1
```

---

I see we could merge it. I would re-base this one. But okay

---

I already closed my PR so that you have less work to do since you'd already included the changes